src/player/PlayerComponent.cpp

```diff
@@ -163,6 +163,11 @@ bool PlayerComponent::componentInitialize()
   mpv::qt::set_property(m_mpv, "hr-seek", "no");
   // Force vo_rpi to fullscreen.
   mpv::qt::set_property(m_mpv, "fullscreen", true);
+  // wait for cache to be filled before starting playback
+  mpv::qt::set_property(m_mpv, "cache-pause-wait", "yes");
```

Comment:
These two feel kinda off-topic; should they be in a separate commit?

Reply:
This is something that allows to wait until cache is filled.
As it was part of the new mpv features, i thought it would be ok for one line :)

Reply:
`cache-pause-wait` is a number; this probably wants `cache-pause-initial`.

Reply:
I think you're right @rcombs; I'll make that change.